data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,609 @@
+	"asteroid scan power" 144
+	"energy consumption" 0.03
+	description "This sensor array features a set of sensitive spectrometers capable of analyzing the spectral signatures of materials in order to determine their composition. A small ultraviolet laser can speed this process up for asteroids by vaporizing a small surface sample, making the spectral lines much more visible, but is sadly useless against shielded ships and their internal systems."
+	description "Unfortunately, while Avgi sensors excel in reading basic physical and chemical information, such as the composition of an object, the amount of sensor data required to predict an outfit's functionality means this sensor can be quite slow when scanning a ship's complex systems or cargo."
+
+
+outfit "Diffuse Deflector"
+	category "Systems"
+	cost 1200000
+	thumbnail "outfit/unknown"
+	"mass" 48
+	"outfit space" -48
+	"delayed shield generation" 0.6
+	"delayed shield energy" 1.8
+	"delayed shield heat" 4.8
+	"shield delay" 3
+	description "For its size, you would expect this shield generator to be far more powerful than it actually is. While it is nearly as large as the heaviest human shield generators, it is only about half as strong, while also consuming much more power. Much of this power ends up being shed as waste heat instead of being used to efficiently regenerate a shield."
+
+
+outfit "Specular Deflector"
+	category "Systems"
+	cost 2100000
+	thumbnail "outfit/unknown"
+	"mass" 72
+	"outfit space" -72
+	"delayed shield generation" 1.2
+	"delayed shield energy" 3.6
+	"delayed shield heat" 9.6
+	"shield delay" 3
+	description "The largest shield generator manufactured by the Avgi is impressive, if only because of how far the Avgi have pushed their limited theory of shielding principles. While their engineering ingenuity is admirable, there is a limit to how far you can take an incomplete understanding of the base physics behind shielding, and this generator approaches that limit."
+
+
+outfit "Optical Lasing Generator"
+	category "Systems"
+	cost 120000
+	thumbnail "outfit/unknown"
+	"mass" 30
+	"outfit space" -30
+	"lasing power" 3
+	description "A monolithic slab of semiconductor diode is used to efficiently pump a spool of doped optical fiber. Hundreds of kilometers long, the coiled fibers serve as a lasing medium with the power needed to produce an intense beam of light, and can be gain-switched to pulse at a variety of frequencies. The produced laser light, which can be converted to a number of wavelengths, can be easily tapped and routed via optical fibers or mirrors to a number of turrets on a ship, eliminating the need for the heavy lasing generator to be anywhere near the turret itself."
+	description "The infrared light produced in the coil can either be passed through a frequency doubling crystal to support up to three Green Optical Lasers, or frequency tripled to support a single Blue Optical Laser."
+
+
+
+# Power:
+
+
+outfit "Concentrating Solar Cell"
```

Comment:
Much better than human solar panels, but weaker than Bunrodea and Coalition ones. Might see some use as a way to fit power generation into small leftover space, which is an option that is nice to have.